node/bft/src/bft.rs

```diff
@@ -613,8 +613,9 @@ impl<N: Network> BFT<N> {
                         // Retrieve the transmission.
                         let Some(transmission) = self.storage().get_transmission(*transmission_id) else {
                             bail!(
-                                "BFT failed to retrieve transmission '{}' from round {}",
+                                "BFT failed to retrieve transmission '{}.{}' from round {}",
                                 fmt_id(transmission_id),
+                                fmt_id(transmission_id.checksum().unwrap_or_default()).dimmed(),
```

Comment:
if [my comment](https://github.com/AleoNet/snarkVM/pull/2521#discussion_r1690551279) on the snarkVM side is implemented (base16 or bech32 encoding), this would need to be updated